recipes/example/meta.yaml

```diff
@@ -21,6 +21,8 @@ source:
 build:
   number: 0
   # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
```

Comment:
Note to others: This was already added in `master` with PR ( https://github.com/conda-forge/staged-recipes/pull/1632 ). So this really isn't an addition, but some issue rendering the diff.